mustermann/lib/mustermann/ast/translator.rb

```diff
@@ -73,6 +73,7 @@ def self.translate(*types, &block)
         Class.new(const_get(:NodeTranslator)) do
           register(*types)
           define_method(:translate, &block)
+          ruby2_keywords :translate
```

Comment:
@jkowens I think you want the if too, otherwise Ruby 2.6 and lower won't be supported? http://eregon.me/blog/2021/02/13/correct-delegation-in-ruby-2-27-3.html, https://github.com/sinatra/sinatra/pull/1750

Reply:
http://eregon.me/blog/2019/11/10/the-delegation-challenge-of-ruby27.html says `ruby2_keywords` is a no-op in Ruby 2.6 but I suspect it doesn't exist before that (I haven't checked though), so that's probably why CI doesn't complain 

(Also haven't checked what Ruby version is required by Sinatra/mustermann)

Reply:
Oh we don't need the `if` because we depend on the `ruby2_keywords` gem https://github.com/sinatra/mustermann/blob/21a0b3ea149befaa9f9bb54353f5738a1ec4ff21/mustermann/mustermann.gemspec#L17